### PR TITLE
Dockerfile for image with latest julia versions and Jupyter 4.2

### PR DESCRIPTION
**Pull request by nkottary**

### CODE REVIEW

File: base/latest/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+To pull a pre-built image from dockerhub, run `docker pull <image>:<version>`
+
+Alternatively, to build locally, run `git clone https://github.com/tanmaykm/JuliaDockerImages.git && docker build -t julialang/julia:latest JuliaDockerImages/base/latest`
+
+## Running
+
+- Run to get a shell prompt: docker run -it <image>:<version>
+- Run Julia: docker run -it --entrypoint="/opt/julia/bin/julia" <image>:<version>
+- Run Jupyter: docker run -it --net="host" --entrypoint="/usr/local/bin/jupyter" <image>:<version> notebook --profile julia
```
Comment (tanmaykm) [inline base/latest/README.md]: Is `--profile julia` still valid?
I get an error: `Unrecognized flag: '--profile'`.
Seems to work without that.

**Comment by tkelman** [comment]
Can we please give all images permanent tag version numbers? latest is not a good idea to be using.

**Comment by tanmaykm** [comment]
This image is meant to have the latest patch versions of all releases.
Maybe tag this as `v0.3-0.6`? Any suggestions?

**Comment by tkelman** [comment]
doesn't really matter what it's called as long as it's archived

**Comment by tkelman** [comment]
something with the date in it might make sense

**Pull request merged**